<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssistantInterface</class>
 <widget class="QWidget" name="AssistantInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QFrame" name="AssistantFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>576</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="ChampionIconLabel">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ChampionNameLabel">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>17</y>
      <width>221</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="ChampionDetailsTabWidget">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>74</y>
      <width>292</width>
      <height>498</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ClosePushButton">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>4</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
